mfr/ext/tabular/render.py

```diff
@@ -35,7 +36,10 @@ def render_html(fp, src=None):
     :return: RenderResult object containing html and assets
     """
 
-    columns, rows = populate_data(fp)
+    try:
+        columns, rows = populate_data(fp)
+    except KeyError:
```

Comment:
I'm thinking this `KeyError` should be handled lower in the call stack; maybe in one of the function in `libs`, or--if that produced lots of duplication--in `populate_data`.